.github/CONTRIBUTING.md

```diff
@@ -1,388 +1,98 @@
 # Contributing
 
-## Requirements
+## Prerequisites
 
-Set up your SSH Key GitHub account and install node.js 4 or higher.
+To contribute to Carbon, you'll need to install:
 
-- [Generating SSH Keys - GitHub](https://help.github.com/articles/generating-ssh-keys/)
-- [`nvm` (Node Version Manager)](https://github.com/creationix/nvm) to use the
-  `Node 6`.
+- [node(v10 or higher)](https://nodejs.org/en/download/)(you can download it
+  here or follow their installation through a package manager
+  [here](https://nodejs.org/en/download/package-manager/))
+  - We recommend using
```

Comment:
```suggestion
  - If you're on macOS, we recommend using [`nvm`](https://github.com/nvm-sh/nvm) to help manage different versions of Node.js 
```

Reply:
I use https://github.com/tj/n, but to each their own.